### PR TITLE
Add ingress to the helm chart

### PR DESCRIPTION
**Pull request by aksakalli**
For some reason #8777 did not include an ingress record in the helm chart for production. [astronomer/airflow-chart](https://github.com/astronomer/airflow-chart/blob/master/templates/ingress.yaml) has an ingress support but it has additional subdomain logic that is not needed for most of the deployments. Thus I adopted the templates from [stable/airflow](https://github.com/helm/charts/blob/master/stable/airflow/templates/ingress-web.yaml).

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by kaxil** [comment]
cc @schnie  if you have time

File: chart/values.yaml [hunk 1]
```diff
@@ -42,6 +42,59 @@ tolerations: []
 # Add common labels to all objects and pods defined in this chart.
 labels: {}
 
+# Ingress configuration
+ingress:
+  # Enable ingress resource
+  enabled: false
```
Comment (alikhtag) [inline chart/values.yaml:48]: Just thinking should this `enabled` conditional moved down to each resource? 
So `ingress.web.enabled` and `ingress.flower.enabled`
I may want `web` enabled, but `flower` ingress disabled if I have `Celery` as executor for example.

**Comment by saikatharryc** [comment]
I have another finding, it would be great to add this same thing in Notes.txt as well

**Comment by alikhtag** [comment]
> I have another finding, it would be great to add this same thing in Notes.txt as well
Indeed this was missing, I've added it! Thanks!

**Review by saikatharryc — APPROVED** [review]

**Comment by saikatharryc** [comment]
anyone has any idea when it can get merged? eagerly waiting for this!

**Review by saikatharryc — APPROVED** [review]

File: chart/values.yaml (hunk 1 above)
Comment (kaxil) [reply chart/values.yaml:48]: @aksakalli WDYT about this suggestion?

Comment (aksakalli) [reply chart/values.yaml:48]: There is no option to disable `flower` if you use `CeleryExecutor` (unlike the [stable/helm](https://github.com/helm/charts/blob/8736171868dd9ba8e73584f3fc486ee7e341718b/stable/airflow/templates/ingress-flower.yaml#L1)). I don't think it is common that someone would like to deploy it with an ingress record for `webserver` but not for `flower`. 

I can make that change if you think that is a necessary use case.

**Review by kaxil — APPROVED** [review]

**Pull request merged**